src/App.js

```diff
@@ -36,7 +50,7 @@ export default function App() {
             element={<AddWishPage onAddDiveWish={addToWishlist} />}
           />
         </Routes>
-      </main>
+      </>
```

Comment:
really nice so far 👍
just one thing...I think the fracment is not necessary at this point! you have already the <AppGrid> as your fracment!?

Reply:
Thx! That was a leftover from trying to fix my Grid :D